felix/bpf-gpl/parsing4.h

```diff
@@ -35,12 +35,7 @@ static CALI_BPF_INLINE int parse_packet_ip_v4(struct cali_tc_ctx *ctx)
 		CALI_DEBUG("ARP: allowing packet\n");
 		goto allow_no_fib;
 	case ETH_P_IPV6:
-		// If IPv6 is supported and enabled, handle the packet
-		if (GLOBAL_FLAGS & CALI_GLOBALS_IPV6_ENABLED) {
-			CALI_DEBUG("IPv6 packet, continue with parsing it.\n");
-			goto ipv6_packet;
-		}
-		// otherwise, drop if the packet is from workload
+		// Drop if the packet is from workload
 		if (CALI_F_WEP) {
 			CALI_DEBUG("IPv6 from workload: drop\n");
```

Comment:
The original comment/code where hot right, works in both directions.

```suggestion
			CALI_DEBUG("IPv6 to/from workload: drop\n");
```